.travis.yml

```
@@ -1,11 +1,37 @@
+dist: xenial
 language: node_js
 node_js:
-  - "0.10"
-sudo: false
-addons:
-  postgresql: 9.3
-before_install:
-- psql -U postgres -c "create database testing_postgis_vt_util"
-- psql -U postgres -d testing_postgis_vt_util -c "create extension postgis"
+- lts/*
+
+sudo: required
+
+env:
+  global:
+    - PGUSER=postgres
+    - PGDATABASE=postgres
+    - PGOPTIONS='-c client_min_messages=NOTICE'
+    - PGPORT=5432
+
+jobs:
+  include:
+    - env: POSTGRESQL_VERSION="9.6" POSTGIS_VERSION="2.5"
+    - env: POSTGRESQL_VERSION="10" POSTGIS_VERSION="2.5"
+    - env: POSTGRESQL_VERSION="11" POSTGIS_VERSION="2.5"
+    - env: POSTGRESQL_VERSION="12" POSTGIS_VERSION="2.5"
+    - env: POSTGRESQL_VERSION="12" POSTGIS_VERSION="3"
+
 script:
-- npm test
+  - sudo service postgresql stop;
```

I wonder if it would make sense to just run all of these in docker containers rather than directly?  Just a thought to make it run faster, but perhaps it is not worth it.

Stop build on the first error:
```suggestion
  - set -e
  - sudo service postgresql stop;
```

---

Being this just for testing, and doing the job of having the different versions of postgres tested and nothing else, I think it's not worth to generate Docker images, but we can always improve this in the future :smile: